src/EditorFeatures/CSharpTest/Diagnostics/SimplifyTypeNames/SimplifyTypeNamesTests.cs

```diff
@@ -3755,7 +3755,6 @@ void M()
     }
 }",
                 options: Option(CodeStyleOptions.QualifyPropertyAccess, false, NotificationOption.Warning),
-                diagnosticCount: 1,
```

Comment:
We can make that change, and it will make such failures easier to diagnose (though it wouldn't take long to figure out the `.Single()` failure). I do suspect there are tons of places where our test helpers do similar things without double-checking first though... Oh well, we can make the world a slightly better place, so let's do it.